data/fields/climbing/grade/yds_class.json

```
@@ -0,0 +1,42 @@
+{
+    "key": "climbing:grade:yds_class",
+    "type": "combo",
+    "label": "Grade (YDS)",
+    "placeholder": "5.10a",
+    "options": [
```

(Missed one with my last change)

```suggestion
    "caseSensitive": false,
    "customValues": true,
    "autoSuggestions": false,
    "options": [
```